doc/nrf/releases/release-notes-2.0.0.rst

```diff
@@ -0,0 +1,1076 @@
+
+  * Updated:
+
+    * Download logic to use the :ref:`lib_dfu_multi_image` library API and image structure.
+    * The image generation script by introducing the sub-element structure inside the Zigbee OTA image.
+      Enable :kconfig:option:`CONFIG_ZIGBEE_FOTA_GENERATE_LEGACY_IMAGE_TYPE` to generate images compatible with previous |NCS| releases.
+    * Default value of the :kconfig:option:`CONFIG_ZIGBEE_FOTA_IMAGE_TYPE` to ``0x0141``.
+    * Setting of the :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` for nRF5340 SoC to ensure integrity of the upgrade image.
+
+  * Removed the :c:enum:`ZIGBEE_FOTA_EVT_ERASE_PENDING` and :c:enum:`ZIGBEE_FOTA_EVT_ERASE_DONE` events.
+
+* Fixed:
+
+  * An issue where printing binding table containing group-binding entries results in corrupted output.
+  * An issue where Zigbee shell coordinator would not form a new network after the factory reset operation.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+* :ref:`crypto`:
+
+  * The Oberon library now supports PKCS#7 padding with CBC and 16 bytes IV with GCM mode in PSA APIs.
+  * Updated:
+
+    * :ref:`nrfxlib:nrf_cc3xx_platform_readme` to v0.9.14.
+    * :ref:`nrfxlib:nrf_cc3xx_mbedcrypto_readme` to v0.9.14.
+    * :ref:`nrfxlib:nrf_oberon_readme` to v3.0.11.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* Added new ``west ncs-sbom`` command that generates :ref:`Software Bill of Materials <west_sbom>`.
+* Added :ref:`bt_fast_pair_provision_script`.
+
+* :ref:`partition_manager`:
+
+  * Added the :file:`ncs/nrf/subsys/partition_manager/pm.yml.pgps` file.
+  * Added the :file:`ncs/nrf/subsys/partition_manager/pm.yml.emds` file.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``e86f575f68fdac2cab1898e0a893c8c6d8fd0fa1``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added :kconfig:option:`CONFIG_NRF53_ENFORCE_IMAGE_VERSION_EQUALITY` to attach the dependencies between application and network core images.
+  This option links the upgrade images in such a way that either both or none is applied.
+* Added support for the Dual-slot execute-in-place (XIP) feature in the |NCS| build system.
+  See the :file:`ncs/nrf/tests/modules/mcuboot/direct_xip` test project for an example of how to leverage this feature in your application.
+* boot_serial: Adapt to Zephyr's new CRC APIs (boot_serial implements the serial recovery),
+* zephyr/boot_serial_extension: use ``BOOT_LOG`` instead of ``LOG_`` (boot_serial_extension implements the serial recovery extension).
```

Comment:
```suggestion
* Modified ``zephyr/boot_serial_extension`` to use ``BOOT_LOG`` instead of ``LOG_`` (``boot_serial_extension`` implements the serial recovery extension).
```